### PR TITLE
Add /content_dashboard/get_content_dashboard_item_info resource command 

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+				).put(
+					"tags", _getAssetTagsJSONArray(contentDashboardItem)
+				).put(
+					"title", contentDashboardItem.getTitle(locale)
+				).put(
+					"userId", contentDashboardItem.getUserId()
+				).put(
+					"userName", contentDashboardItem.getUserName()
+				).put(
+					"userPortraitULR",
```
Comment (carloslancha) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java:137]: Typo `ULR`

**Comment by carloslancha** [comment]
@cgoncas creationDate is not being passed (don't know if that's intended, but it's on the designs)
image

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java (hunk 1 above)
Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java:137]: Thanks!

**Comment by cgoncas** [comment]
Hey @carloslancha 

 I have updated this pull with the rename of userPortraitURL, the addition of createDate and languageTag.

:)

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**